platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/etc/init.d/i2c_init

```diff
@@ -188,6 +204,7 @@ load_i2c_dev_modules () {
 case "$1" in
 start)
     echo -n "Setting up board... "
+    depmod -a
```

Comment:
depmod -a should not be in init script, it should be postinst for package installation.

Reply:
We create an environment to test reboot 1000 times. If we don't add "depmod -a" here, we will get a "libkmod: ERROR" message.

According to "https://books.google.com.tw/books?id=hG_bhCL4tQ8C&pg=PA417&lpg=PA417&dq=why+depmod+everytime&source=bl&ots=DPSB8IGh3M&sig=CU8KApYOLwEeTXYGYpI5LtDIQtY&hl=zh-TW&sa=X&ved=2ahUKEwjXhcuh9pjbAhXBU7wKHee9BQoQ6AEwBXoECAEQZg#v=onepage&q=why%20depmod%20everytime&f=false", "depmod -a" should run every boot.

Reply:
I cannot see the book. depmod should not be in init script, it should be only run during package installation.

Reply:
Please go to the directory in ${sonic}/platform/broadcom/ and execute following command

grep -nri --exclude-dir=debian --exclude=*.[ch] "depmod -a" *

You can see not every vendor runs "depmod -a" once during package installation.  Some vendors run "depmod -a" during every boot via *.py which is executed by *.service of systemd.

Reply:
![image](https://user-images.githubusercontent.com/35956037/44648774-b5257980-aa14-11e8-80df-0e6b7327a24c.png)

Reply:
modeles.dep does not need to be re-created every boot. we need to re-build them when we introduce new kernel moduels into the image. essentially the package installation time. As you can see, not all vendors are doing the right thing, we will correct them one-by-one, and we can discuss this in the coming community meeting.

Reply:
I will remove "depmod -a" from init script.